packages/contracts-bedrock/package.json

```diff
@@ -50,6 +50,7 @@
   },
   "devDependencies": {
     "@eth-optimism/hardhat-deploy-config": "^0.2.0",
+    "@eth-optimism/contracts-periphery": "^0.1.5",
```

Comment:
I would prefer that the dependency chain go from `contracts-bedrock` to `contracts-periphery`. I'm fine with moving `Semver` into `contracts-bedrock` to achieve this. I just don't want a circular dependency and I can imagine building peripheral contracts that interop with some of our core contracts.